crates/ruff/src/rules/pycodestyle/helpers.rs

```diff
@@ -56,3 +57,52 @@ pub fn is_overlong(
 
     true
 }
+
+pub fn is_keyword_token(token: &Tok) -> bool {
+    match token {
```

Comment:
@charliermarsh I noticed that we have `KWLIST` but converting `Tok` to a string would require a hacky solution (they add single quotes to most tokens so we would need to remove them, eg `"')'"`) so I added these helper methods.